spec/index.bs

```diff
@@ -7172,6 +7185,14 @@ run the following steps:
 
                     1. Return `null` (which will cause the creation of the pipeline to fail).
 
+                1. If |resource| is a storage texture binding,
+                    |entry|.|storageTexture|.{{GPUStorageTextureBindingLayout/access}} is {{GPUStorageTextureAccess/"read-write"}},
+                    |previousEntry|.|storageTexture|.{{GPUStorageTextureBindingLayout/access}} is {{GPUStorageTextureAccess/"write-only"}}, and
+                    |previousEntry|.|storageTexture|.{{GPUStorageTextureBindingLayout/format}} is compatible with
+                    {{GPUTextureUsage/STORAGE_BINDING}} and {{GPUStorageTextureAccess/"read-write"}} according to the [[#plain-color-formats]] table:
```

Comment:
IIUC this only upgrades WO to RW, but won't upgrade RO to RW. Shouldn't that also be possible? @Kangz

Reply:
No that's not possible unfortunately, on D3D12 for example the RO storage textures use SRVs while RW/WO use UAVs.

Reply:
Ah, I missed that we can use SRVs for RO storage textures.

My analysis in https://github.com/gpuweb/gpuweb/issues/3838#issuecomment-1761399085 is not correct as I was looking for "UAV Typed Load" instead of "Shader ld" in https://learn.microsoft.com/en-us/windows/win32/direct3ddxgi/format-support-for-direct3d-11-0-feature-level-hardware.

I'll check again for the formats that support "Shader ld", they'll most likely be at least those that support `STORAGE_BINDING` usage (currently listed in the spec) which would allow us to then move the read-only access together with the write-only access in the capability table.

Reply:
I confirm that all formats that support "UAV Typed Store" (which corresponds to our `STORAGE_BINDING` usage) also support "Shader ld".

We can move the read-only access to the original `STORAGE_BINDING` column in the table.

Reply:
@alan-baker re 60644c8b8a11a79f4baa3ac58086cfd8a5ddab60, we still need the read-write access to be separate in its own column.

Reply:
Ok, I reverted that and basically moved "read-only" to the "write-only" column.